tests/conftest.py

```diff
@@ -199,6 +209,23 @@ def _cleanup(ec2, elb, resource_name_prefix: str):
         elb.delete_load_balancer(LoadBalancerName=lb_name)
 
 
+def _cleanup(ec2, elb, resource_name_prefix: str):
+    functions = [
+        (cleanup_instances, (ec2, resource_name_prefix)),
+        (cleanup_volumes, (ec2, resource_name_prefix)),
+        (cleanup_igw, (ec2, resource_name_prefix)),
+        (cleanup_vpc, (ec2, resource_name_prefix)),
+        (cleanup_ssh, (ec2, resource_name_prefix)),
+        (cleanup_elb, (elb, resource_name_prefix)),
+    ]
+    # ignore cleanup failures
+    for function, arguments in functions:
+        try:
+            function(*arguments)
+        except Exception:
+            LOGGER.exception("Failed to execute cleanup method %s", function.__name__)
```

Comment:
Shouldn't we print the exception for debugging purposes.

Reply:
it will (or should) (https://docs.python.org/3/library/logging.html#logging.Logger.exception)

Reply:
True. I missed the fact that it was logged as exception.